src/main/java/org/folio/des/controller/FolioTenantController.java

```diff
@@ -60,4 +109,299 @@ public ResponseEntity<Void> postTenant(TenantAttributes tenantAttributes) {
     return tenantInit;
   }
 
+  public void recreateLegacyJobs() {
```

Comment:
@GoZaddy could we move this in there too?

Reply:
For sure!

Reply:
@ncovercash done!